src/core/openthread.cpp

```diff
@@ -1636,6 +1636,21 @@ ThreadError otSetActiveDataset(otInstance *aInstance, const otOperationalDataset
     return error;
 }
 
+bool otIsNodeCommissioned(otInstance *aInstance)
+{
+    otOperationalDataset aDataset;
+
+    otGetActiveDataset(aInstance, &aDataset);
```

Comment:
I'm taking a look at the error now. Though calling directly does seem like a good solution.